recipe/meta.yaml

```diff
@@ -28,7 +28,8 @@ source:
     - 0016-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
 
 build:
-  number: 1
+  skip: true  # [win]
```

Comment:
Note [Windows is already disabled in `main`]( https://github.com/conda-forge/r-base-feedstock/blob/6a79d65c6813758360bd884020ac1106471cf470/recipe/meta.yaml#L31 ) due to issue ( https://github.com/conda-forge/r-base-feedstock/issues/214 ). Given this issue would be seen with any Windows build of `r-base`, went ahead and just disabled here until that issue can be resolved.

Reply:
No, windows is disabled on main, because we don't have ucrt based mingw compilers which is needed for R>=4.2.0. For R<4.2.0, the mingw packages are just fine.

Reply:
Issue ( https://github.com/conda-forge/r-base-feedstock/issues/214 ) shows an expired certificate problem. Don't think this is UCRT related, but maybe I'm missing something.

Reply:
Yes, you are missing something. Please read https://github.com/conda-forge/r-base-feedstock/pull/205